src/Raven.Studio/typescript/widgets/virtualGrid/checkedColumn.ts

```diff
@@ -0,0 +1,26 @@
+﻿import virtualColumn = require("widgets/virtualGrid/virtualColumn");
+
+/**
+ * Checked cell template used for toggling selection of a row in the virtual grid.
+ */
+class checkedColumn implements virtualColumn {
+    public readonly cellClass = "checked-cell";
```

Comment:
Sure, I'll remove it. p.s. I agree with Resharper we should eventually move class names to start with upper case. That seems to be the emerging standard for TypeScript anyways.